Editor/ReactiveObjects/ParameterAssignerPass.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.Collections.Generic;
+﻿#if MA_VRCSDK3_AVATARS
```

Comment:
It might be cleaner to break out the ReactiveObjects directory into a different assembly and conditionally compile it...